core/python2Action/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for Python 2 actions, mirrors the Python 3-based pythonAction
+
+FROM python:2.7.13-alpine
```

Comment:
The [idea](https://github.com/openwhisk/openwhisk/issues/1184#issuecomment-285323326) here is to make dockerskeleton into a **Python 3** container.  The plan is to migrate both the docker images core/actionProxy (aka dockerskeleton) and core/pythonAction from Python 2.7 to Python 3.6.  The new core/python2Action image is meant to be our _temporary_ method of continuing to run Python 2 jobs after that transition to Python 3.  The alternative was to create a single core/actionProxy that contains _both_ Python 2 and Python 3 in a single image.  That [was tried -- it works -- but was rejected](https://github.com/openwhisk/openwhisk/pull/1903#issuecomment-284717687) because the resulting container is too big for daily use.